static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+
+import {AreaChart, AreaChartProps} from 'sentry/components/charts/areaChart';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {formatBytesBase2} from 'sentry/utils';
+import theme from 'sentry/utils/theme';
+
+function MemoryChart({memorySpans}) {
+  const chartOptions: Omit<AreaChartProps, 'series'> = {
+    grid: {
+      left: '10px',
+      right: '10px',
+      top: '40px',
+      bottom: '10px',
+    },
```

Comment:
what does this `grid` setting do? 

I would've guessed gridlines on the chart, horizontal or vertical, but i'm not seeing any of those... and left/right/top/bottom don't correlate to that idea either. So i'm lost.